arrow/src/test/java/org/apache/iceberg/arrow/vectorized/TestArrowReader.java

```diff
@@ -722,11 +770,22 @@ private void writeTable(boolean constantRecords) throws Exception {
             Types.NestedField.required(26, "decimal", Types.DecimalType.of(9, 2)),
             Types.NestedField.optional(27, "decimal_nullable", Types.DecimalType.of(9, 2)),
             Types.NestedField.required(28, "fixed", Types.FixedType.ofLength(7)),
-            Types.NestedField.optional(29, "fixed_nullable", Types.FixedType.ofLength(7)));
+            Types.NestedField.optional(29, "fixed_nullable", Types.FixedType.ofLength(7)),
+            Types.NestedField.required(30, "timestamp_nano", Types.TimestampNanoType.withoutZone()),
+            Types.NestedField.optional(
+                31, "timestamp_nano_nullable", Types.TimestampNanoType.withoutZone()),
+            Types.NestedField.required(32, "timestamp_nano_tz", Types.TimestampNanoType.withZone()),
+            Types.NestedField.optional(
+                33, "timestamp_nano_tz_nullable", Types.TimestampNanoType.withZone()));
 
     PartitionSpec spec = PartitionSpec.builderFor(schema).month("timestamp").build();
 
-    Table table = tables.create(schema, spec, tableLocation);
+    Table table =
+        tables.create(
+            schema,
+            spec,
+            ImmutableMap.of(TableProperties.FORMAT_VERSION, "3"),
+            tableLocation); // .create(schema, spec, tableLocation);
```

Comment:
the comment can be removed now

Reply:
Ooops right, that's a leftover comment there. Thanks for spotting it. Is it okay to change the format version in this test?